### ISSUE TITLE
Screenreader can't open drop-down-menu (b-form-select)

### ISSUE THREAD

**Issue by Distaly** [LAYAIraq/LAYA-Iraq-Frontend#287]
Opening the 'change content type' in the courses opens a modal window. In this modal, the screenreader will read that it has found a drop-down-menu, however it is unable to open it.

Drop-down-menu can't be accessed using the arrow buttons (but is announced). Can be accessed using the 'tab' key.
